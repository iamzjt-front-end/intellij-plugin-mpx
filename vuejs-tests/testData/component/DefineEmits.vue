<script setup>
defineEmits(["test-event"])
</script>
