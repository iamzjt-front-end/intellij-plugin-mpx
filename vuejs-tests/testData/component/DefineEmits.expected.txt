{
 class: "VueSourceComponent",
 components: {
  DefineEmits: <recursive value of class VueSourceComponent>,
 },
 emits: [
  {
   class: "VueScriptSetupLiteralBasedEvent",
   name: "test-event",
  },
 ],
}
